.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+	"useTabs": true,
+	"singleQuote": false,
+	"semi": true,
+	"endOfLine": "lf",
+	"arrowParens": "always",
+	"printWidth": 100
+}
```

Comment:
Can we add a newline to the end of this file?

Reply:
Oops yeah when I reverted the formatting commit with a force push I undid that too!